recipes/tracy/all/conanfile.py

```diff
@@ -157,17 +157,16 @@ def package_info(self):
         if self.options.get_safe("libunwind_backtrace"):
             self.cpp_info.components["tracyclient"].requires.append("libunwind::libunwind")
 
+        # Starting at 0.12.0, upstream has added an extra "tracy" directory for the include directory
+        # include/tracy/tracy/Tracy.hpp
+        # but upstream still generates info for including headers as #include <tracy/Tracy.hpp>
+        if Version(self.version) >= "0.12.0":
+            self.cpp_info.components.includedirs = ['include/tracy']
+
         # Tracy CMake adds options set to ON as public
         for opt in self._tracy_options.keys():
             switch = getattr(self.options, opt)
             opt = f"TRACY_{opt.upper()}"
             if switch:
                 self.cpp_info.components["tracyclient"].defines.append(opt)
 
-        # TODO: to remove in conan v2
-        self.cpp_info.names["cmake_find_package"] = "Tracy"
-        self.cpp_info.names["cmake_find_package_multi"] = "Tracy"
-        self.cpp_info.components["tracyclient"].names["cmake_find_package"] = "TracyClient"
```

Comment:
This introduced this regression here: https://github.com/conan-io/conan-center-index/pull/27888 

Apologies as we should have picked this up during the review process. We've seen this in other recipes, that lines below the "TODO remove in Conan v2" .. some of those lines should not be removed. 

Another reminder that we are asking contributors to limit the PR to the minimal changes required to fix their issue (https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md#pull-request-scope), and reviewers to enforce this :(